### PR TITLE
Add Datelist connector 

### PR DESCRIPTION
**Pull request by aclarembeau**
# Datelist - Microsoft Flow Connector
Microsoft flow connector for https://datelist.io 
Datelist is an online booking system platform that allows customers taking meetings online. 

## Triggers

- New booking: Triggers when someone books a meeting in the Datelist platform 

## Actions

- Get calendars: List all available calendars in Datelist.io 

# Questions 
I've tried to make our organization name as the Stack owner (datelist.io), but I get the following error: 
> The request content was invalid and could not be deserialized: 'Could not find member 'stackOwner' on object of type 'ApiPropertiesDefinition'. Path 'properties.stackOwner', line 1, position 384.'. 

So, I've put my personal name here "Clarembeau Alexis", do you know what are the accepted values in that field? 

# Checks done
Please check the following conditions for your PR.

- [X] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [X] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

**Review by sriyen-msft — APPROVED** [review]

File: custom-connectors/Datelist/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#818BE1",
+    "capabilities": [],
+    "publisher": "Alexis Clarembeau"
```
Comment (sriyen-msft) [inline custom-connectors/Datelist/apiProperties.json:20]: @aclarembeau Please add `stackOwner` property as well

Comment (dougelliott) [inline custom-connectors/Datelist/apiProperties.json:20]: We'd recommend that the `publisher` and `stackOwner` properties do not contain PII. It's better for it to be representative of the company as a whole.

**Review by dougelliott — CHANGES_REQUESTED** [review]

**Comment by dougelliott** [comment]
@aclarembeau Could you please also sign the CLA? Thanks!

**Comment by aclarembeau** [comment]
Hello, and thank you for the feedbacks. 
I've signed the CLA, removed PII and used ACL Software (our company) as a publisher and a stackowner. 
Is that good with those modifications?

File: custom-connectors/Datelist/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+      "propertyValue": "https://datelist.io/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Communication;Productivity"
+    }
+  ],
+  "security": [
+    {
+      "Cl\u00e9 API": []
```
Comment (sriyen-msft) [inline custom-connectors/Datelist/apiDefinition.swagger.json:300]: @aclarembeau Unicodes are not allowed in any connector's artifact, Can you please remove them all.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: custom-connectors/Datelist/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Datelist - Microsoft Flow Connector
+Microsoft flow connector for https://datelist.io
```
Comment (sriyen-msft) [inline custom-connectors/Datelist/README.md:2]: @aclarembeau `Flow` is an outdated word in the product, Can you please update is with `Power Automate`

File: custom-connectors/Datelist/apiDefinition.swagger.json (hunk 2 above)
Comment (sriyen-msft) [reply custom-connectors/Datelist/apiDefinition.swagger.json:300]: @aclarembeau Any update ?

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by sriyen-msft** [comment]
@aclarembeau I'm just checking in to see how things are going with updating your connector as per the feedback. Feel free to reach out with any questions you may have.

**Comment by aclarembeau** [comment]
Hello @sriyen-msft , sorry for the delay. I've applied your recommendations; so, it's good on my side.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**